### PR TITLE
Add ownership validation table schema option

### PR DESCRIPTION
**Pull request by zhangn49**
## Overview

Description:

Added ownership validation table schema option

Why should this be merged: 

Related issue(s) (if applicable): #<number>

## Checklist (Definition of Done):

- [x] There are no TODOs left in the code
- [x] [Coding conventions](https://github.com/CorfuDB/CorfuDB/wiki/Corfu-Style-Guidelines) (e.g. for logging, unit tests) have been followed
- [x] Change is covered by automated tests
- [x] Public API has Javadoc

### CODE REVIEW

**Review by WenbinZhu — APPROVED** [review]

File: runtime/proto/corfu_options.proto [hunk 1]
```diff
@@ -19,6 +19,8 @@ message SchemaOptions {
     repeated string stream_tag = 5;
     // Nested Secondary Key (repeated field)
     repeated NestedSecondaryIndex nested_secondary_key = 6;
+    // Should the entry of table be validated ownership
```
Comment (pankti-m) [inline runtime/proto/corfu_options.proto:22]: this comment is a bit unclear... what does the option mean?

Comment (zhangn49) [reply runtime/proto/corfu_options.proto:22]: his flag will indicate whether the shim-layer will make a call to validate the ownership of the DB entry.

Comment (pankti-m) [reply runtime/proto/corfu_options.proto:22]: ok, can we say something like 'should the entry.. validated for ownership'?

Comment (zhangn49) [reply runtime/proto/corfu_options.proto:22]: sure, will do

**Review by pankti-m — APPROVED** [review]

File: runtime/proto/corfu_options.proto (hunk 1 above)
Comment (zhangn49) [reply runtime/proto/corfu_options.proto]: Done

**Review by annym — APPROVED** [review]

**Pull request merged**